modules/ROOT/pages/my-applications.adoc

```diff
@@ -47,12 +47,13 @@ Lists the application’s redirect URIs.
 Lists the application’s grant types. 
 * API Contracts
 +
-Lists the application’s API name, environment and instances, SLA tier, and request access state. The environment can be either a sandbox or development environment. Request access states can be `pending, `approved, or `failed.
+From the API contracts section, you can see the application’s API name, environment and instances, instance URLs, SLA tier, and request access state. The environment can be either a sandbox or development environment. Request access states can be `pending, `approved, or `failed.
```

Comment:
Is this necessary - describing the UI? I think the bullet list below is probably more useful because it covers the tasks that the user wants to do.